src/PowerSystems.jl

```diff
@@ -540,7 +540,9 @@ import InfrastructureSystems:
     is_convex,
     get_points,
     get_x_coords,
```

Comment:
We need a better name for this to avoid concept clash with the geographical information. Something to the effect that this is for pwl should be clear in the function name

Reply:
Left as TODO for now